src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/31.diff.sql

```diff
@@ -0,0 +1,13 @@
+﻿/*************************************************************
+    This migration adds new constraing on data length of rawresource
+**************************************************************/
+
+ALTER TABLE dbo.Resource WITH NOCHECK
+    ADD CONSTRAINT RawResourceLength CHECK (DATALENGTH(RawResource) > 49)
```

Comment:
We might need to check if the object already exists or not before adding a constraint. SQL test will fail where we try to apply same diff script twice

Reply:
Check for exists added.

Reply:
Should we add check like below instead of dropping it if exists?
IF NOT EXISTS (SELECT * FROM sys.objects WHERE name = N'RawResourceLength' AND type = N'C')
THEN <ADD CONSTRAINT>